Sources/PostgresNIO/Data/PostgresDataType.swift

```diff
@@ -16,7 +16,7 @@ public enum PostgresFormatCode: Int16, Codable, CustomStringConvertible {
 
 /// The data type's raw object ID.
 /// Use `select * from pg_type where oid = <idhere>;` to lookup more information.
-public struct PostgresDataType: Codable, Equatable, ExpressibleByIntegerLiteral, CustomStringConvertible, RawRepresentable {
```

Comment:
Note: I have moved the Codable and ExpressibleByIntegerLiteral further down and added a remove `TODO` for the next major version break.